sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -20,7 +20,7 @@ autorest --android --use=C:/work/autorest.java
 
 ### Code generation settings
 ``` yaml
-input-file: https://int.chatgateway.trafficmanager.net/swagger/2020-11-01-preview3/swagger.json
+input-file: swagger.json
```

Comment:
It needs to be a url. Where does this swagger file come from, is it preview4?

Reply:
The swagger file is checked in to this PR. It is a temporary file and you can delete it and replace it with the preview4 swagger once it is ready.